README.adoc

```diff
@@ -63,7 +63,7 @@ https://asciinema.org/a/wVkVgUrO7PGR5CYBFbHB5fFDn[image:https://asciinema.org/a/
 
 *Chat:* Join us on link:https://join.slack.com/t/openshiftdo/shared_invite/enQtODUwMjIzMzE2MjYyLWM3YjlkNTViOGNjZGQ5YjFlMzc2N2RhZjVmMDQ5NWY5ZDQ3NzMzZjYxMTE1MjY5OTVmZjgxZWMyMGRmOTc4Y2I[Slack].
 
-*Discussions:* For any developmental discussions with `odo`, use our link:https://groups.google.com/forum/#!forum/odo-dev[odo-dev Google group], for general help and inquiries, join our link:https://groups.google.com/forum/#!forum/odo-users[odo-users Google group].
+*Discussions:* For any developmental discussions with `odo`, use our link:https://groups.google.com/forum/\#!forum/odo-dev[odo-dev Google group], for general help and inquiries, join our link:https://groups.google.com/forum/\#!forum/odo-users[odo-users Google group].
```

Comment:
This is really really odd.. First link works, second link doesn't, even though they're the exact same. GitHub asciidoc bug perhaps?